flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.html

```diff
@@ -45,12 +45,13 @@
   >
     <thead>
       <tr>
-        <th nzWidth="60px"></th>
+        <th nzWidth="70px"></th>
         <th><strong>Name</strong></th>
         <th><strong>Acknowledged</strong></th>
         <th><strong>Latest Acknowledgment</strong></th>
         <th><strong>End to End Duration</strong></th>
-        <th><strong>Checkpointed Data Size</strong></th>
+        <th><strong>Incremental Checkpoint Data Size</strong></th>
+        <th><strong>Full Checkpoint Data Size</strong></th>
```

Comment:
WDYT about adding a tooltip here and/or in other added tags? (maybe copying the javadoc).
I think `title` attribute should work.

Reply:
Agree, I will add a tooltip.